ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.13-nightly-x86.yaml

```diff
@@ -0,0 +1,129 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-rosa-hcp-4.13-nightly
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-hcp-3zones-control-plane-24nodes-periodic
+  cron: 0 12 * * 4
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: rosa-aws-sts-hypershift
+  timeout: 5h0m0s
+- always_run: false
+  as: perfscale-rosa-hcp-1zone-cluster-density-v2-3nodes
```

Comment:
I thought we were removing 1zone control plane tests.

Reply:
@afcollins Removed single zone cluster density tests and multi zone data plane tests for HCP as discussed in Jedi group. Also, added single zone data plane missing tests for HCP.
I'll remove remaining "single zone cluster density tests and multi zone data plane tests " for remaining configs in a separate PR as I am not touching those other configs here.